website/sidebars.js

```diff
@@ -13,130 +13,75 @@
 
 /** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
 const sidebars = {
-  // By default, Docusaurus generates a sidebar from the docs folder structure
-  ///tutorialSidebar: [{ type: 'autogenerated', dirName: '.' }],
-
-  // But you can create a sidebar manually
-
+  // Automatically generate sidebar from the docs folder structure
+  // This allows the sidebar to reflect the new organized directory layout
   tutorialSidebar: [
```

Comment:
Instead of listing outa each category and making the autogenerated, let's make the whole sidebar autogenerated as shown here: https://docusaurus.io/docs/2.x/sidebar/autogenerated (look for "Docusaurus can generate a full sidebar from your docs folder")